file-formats/zif_aff_types_v1.intf.abap

```diff
@@ -114,6 +114,17 @@ INTERFACE zif_aff_types_v1 PUBLIC.
       abap_language_version TYPE ty_abap_language_version,
     END OF ty_header_60.
 
+  TYPES:
+    "! <p class="shorttext">Header</p>
+    "! The header for an ABAP main object (without source code; without ABAP language  version)
```

Comment:
```suggestion
    "! The header for an ABAP main object (without source code; without ABAP language version)
```

Reply:
My local abaplint in VS Code didn't reported this :D 
I guess I need to check it.